NDLANO/editorial-frontend
# 2563 redigering av custom fields i taxonomy

Fixes https://github.com/NDLANO/Issues/issues/2563

How to test: 
1) Visit /structure
2) Choose a subject and click on the gear icon
3) Click on `Legg til/endre metadata` button
4) Check that u can add and remove fields
5) Do the same thing for topics
